app/Util/Media/Image.php

```diff
@@ -157,9 +159,11 @@ public function handleImageTransform(Media $media, $thumbnail = false)
 					$media->metadata = json_encode($meta);
 				}
 
-				$img->resize($aspect['width'], $aspect['height'], function ($constraint) {
-					$constraint->aspectRatio();
-				});
+				if ( ($ratio['width_original'] > $aspect['width']) || ($ratio['height_original'] > $aspect['height']) ) {
```

Comment:
```suggestion
				if (
				    ($ratio['width_original'] > $aspect['width'])
				    || ($ratio['height_original'] > $aspect['height'])
				) {
```

Not sure on the styling guidelines, but I think this makes it a bit more readable. Happy for suggestion to be closed off if pixelfed are happy :+1: